tests/tests/website_footer_tests.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Documentation  Smoke Tests
+Library        OperatingSystem
+Library        SeleniumLibrary
+Library        String
+Resource       ../src/page_objects/keywords/common.robot
+Resource       ../src/page_objects/keywords/home_page.robot
+Resource       ../src/page_objects/keywords/ubuntu.robot
+Resource       ../src/test_files/test_files.resource
+Library        ../src/page_objects/libraries/browser_lib.py
+Suite Setup   Prepare Test Environment
+Test Setup    Prepare For Test
+Test Teardown    Clean After Test
+
+
+*** Test Cases ***
+Find and open About page
+    [documentation]  Click button About and check if subpage is opened
+    [tags]  Regression
+    #TODO: Find button About, clicks it and verifies if a list with some text dislays
+    Click Element  ${AboutButton}
+    Wait Until Element Is Visible   ${AboutSubpageText}    1
+
+Find and open Known Issues page
+    [documentation]  Click button Known Issues and check if subpage is opened
+    [tags]  Regression
+    #TODO: Find button Known Issues, clicks it and verifies if a list with some text dislays
+    Click Element  ${KnownIssuesButton}
+    Wait Until Element Is Visible  //div[@id="issues"]/ul/*
```

Comment:
Maybe check if solution from the comment above is applicable to this case here.
Move locator to locators.py and have a variable here.